generator/templates/Crud/src/VuetifyResourceTexts.js

```diff
@@ -13,5 +16,8 @@ export default {
   'snackbar-get-error': 'Er ging iets mis, probeer het later opnieuwr',
   'snackbar-deleted': 'Het is verwijderd!',
   'snackbar-delete-error': 'Er ging iets mis, probeer het later opnieuw',
-  'search': 'Zoeken',
+  search: 'Zoeken',
+  loading: 'Laden...',
+  'delete-confirmation-title': 'Weet u het zeker?',
+  'delete-confirmation-content': 'Weet u het zeker dat u deze item(s) wilt verwijderen?',
```

Comment:
Is this correct spelled Dutch? 😅

Reply:
This is being used in one of our projects
``` 
   "deleteConfirmation": {
        "title": "Weet je het zeker?",
        "body": "Je staat op het punt om dit {resource} te verwijderen, weet je zeker dat je door wilt gaan?"
    },
```